### ISSUE TITLE
[Core] Notifications are below the main window of FreeCAD

### ISSUE THREAD

**Issue by donovaly** [FreeCAD/FreeCAD#8940]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Forums discussion

https://forum.freecad.org/viewtopic.php?p=668153#p668153

### Version

0.21

### Full version info

```shell
OS: Windows 10 Version 2009
Word size of FreeCAD: 64-bit
Version: 0.21.0.32357 (Git)
Build type: Release
Branch: master
Hash: d23b576c3361b7d593b8e65255a5f970e5954c39
Python 3.8.10, Qt 5.15.2, Coin 4.0.1, Vtk 8.2.0, OCC 7.6.3
Locale: English/United States (en_US) [ OS: German/Germany (de_DE) ]
```

### Subproject(s) affected?

Core

### Issue description

The message notification feature shows messages always below FreeCAD's main window.

Here is a screencast of a selection mistake. The error message appears below the main window:
![5YaSOFhAF0](https://user-images.githubusercontent.com/1828501/226076551-7589aeec-c90a-4277-97a9-db6c29571e4e.gif)

### Anything else?

@abdullahtahiriyo 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by donovaly** [comment]
@abdullahtahiriyo

**Comment by abdullahtahiriyo** [comment]
I see.

In my PC this renders inside the frame. I will look into it.

**Comment by donovaly** [comment]
Will be fixed with #8958
